src/DependencyInjection/Configuration.php

```diff
@@ -134,6 +134,26 @@ public function getConfigTreeBuilder()
                         ->booleanNode('sort_admins')->defaultFalse()->info('Auto order groups and admins by label or id')->end()
                         ->booleanNode('confirm_exit')->defaultTrue()->end()
                         ->booleanNode('js_debug')->defaultFalse()->end()
+                        ->scalarNode('skin')
```

Comment:
What about using [enumNode](https://symfony.com/doc/4.4/components/config/definition.html#enum-nodes), so there is no need of manual validation.